<launch>
	<!-- Hand to eye calibration node -->
	<node pkg="rovi2_development" type="hand_to_eye" name="hand_to_eye_calibration" output="screen">
		<param name="image_left" value="/camera/left/image_raw" />
		<param name="image_right" value="/camera/right/image_raw" />
		<param name="sub_robot_state" value="/ur_simple_demo_node/caros_serial_device_service_interface/robot_state" />
		<param name="calibration_yaml_path_left" value="left_front.yaml" />
		<param name="calibration_yaml_path_right" value="right_front.yaml" />

		<param name="pub_robot_state" value="/q_state_processed" />
		<param name="pub_point_left" value="/pose/2d_left" />
		<param name="pub_point_right" value="/pose/2d_right" />

	</node>
</launch>
